extensions/objects/templates/list-of-sets-of-translatable-html-content-ids-editor.component.ts

```diff
@@ -82,14 +82,9 @@ export class ListOfSetsOfTranslatableHtmlContentIdsEditorComponent {
       this.valueChanged.emit(this.value);
     }
     // Removing any empty arrays from the end.
-    for (let i = 1; i < this.value.length; i++) {
-      if (this.value[i].length === 0) {
-        // If empty array is found, all subsequent arrays must also be
-        // empty since rank skipping is not allowed.
-        this.value = this.value.slice(0, i);
-        this.valueChanged.emit(this.value);
```

Comment:
Is this emit statement no longer required?

Reply:
Oh, I missed this. I've readded this, thanks!